spec/system/candidate_interface/candidate_entering_personal_details_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
+require_relative 'helpers/candidate_helper'
```

Comment:
`spec/helpers` are specs for the view helpers, this file probably should go into `spec/support` (which is auto-loaded so you don't need the `require`).